apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -74,14 +94,31 @@ export function InstructorCourseAdminSettings({
                 </div>
                 <div class="form-group">
                   <label for="display_timezone">Timezone</label>
-                  <input
-                    type="text"
-                    class="form-control"
-                    id="display_timezone"
-                    name="display_timezone"
-                    value="${resLocals.course.display_timezone}"
-                    disabled
-                  />
+                  <select 
+                    class="form-control" 
+                    id="display_timezone" 
+                    name="display_timezone" 
+                    ${courseInfoExists && resLocals.authz_data.has_course_permission_edit ? '' : 'disabled'}
+                  >
+                    ${availableTimezones.map(
+                      (tz, i) => html`
+                        <option
+                          value="${tz.name}"
+                          id="timezone-${i}"
```

Comment:
Do these need IDs?

Reply:
No. Removed in [79f4f05](https://github.com/PrairieLearn/PrairieLearn/pull/9581/commits/79f4f05b8d78db39a94b29e232a22bdc4c552b34).